Pspcl/Pspcl.Web/Views/Shared/NoSideBarLayout.cshtml

```diff
@@ -0,0 +1,50 @@
+﻿<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>@ViewData["Title"]</title>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-GLhlTQ8iRABdZLl6O3oVMWSktQOp6b7In1Zl3/Jr59b6EGGoI1aFkw7cmDA6j6gD" crossorigin="anonymous">
+    <link rel="stylesheet" href="~/css/site.css" asp-append-version="true" />
+    <link rel="stylesheet" href="~/Pspcl.Web.styles.css" asp-append-version="true" />
+    @RenderSection("styles", false)
+</head>
+<body>
+    
+    <header>
+        <nav class="navbar navbar-expand-sm navbar-toggleable-sm bg-dark border-bottom box-shadow mb-3">
+            <div class="container">
+                <a class="navbar-brand" asp-area="" asp-controller="Home" asp-action="Index"><img src="https://upload.wikimedia.org/wikipedia/commons/5/54/PSPCL_Logo.png" alt="logo" height="70px" class="bg-white"></a>
+                <div class="navbar-collapse collapse d-sm-inline-flex d-flex justify-content-center text-white">
+                    <h2>Punjab State Power Corporation Limited</h2>
+                </div>
+            </div>
+        </nav>
+    </header>
+
+    <div class="container">
+        <main role="main" class="pb-3">
+            @RenderBody()
+        </main>
+    </div>
+
+
+    <footer class="border-top footer text-muted mt-5">
+        <div class="container">
+            <p><center>Powered By LnGConsultancy</center></p>
+        </div>
+    </footer>
+
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/js/bootstrap.bundle.min.js" integrity="sha384-w76AqPfDkMBDXo30jS1Sgez6pr3x5MlQ1ZAGC+nuZB+EYdgRZgiwxhTBTkF7CXvN" crossorigin="anonymous"></script>
```

Comment:
2 bootstrap jquery files reference given. The order of files is also wrong. Jquery library should come first, then bootstrap. You can remove bootstrap bundle JS and use bootstrap.min.

Reply:
Resolved now. following is the order now:  **jquery-3.5.1.min.js,  popper.min.js,  bootstrap.min.js**

Reply:
popper min should be after bootstrap. Do confirm from your research also.

Reply:
i researched , and used chatGpt also, it said the same order